Sources/TuistGraph/Models/WorkspaceGenerationOptions.swift

```diff
@@ -51,14 +51,19 @@ extension Workspace {
         /// It is recommended to set the version option to Xcode's version that is used for development of a project, for example `.lastXcodeUpgradeCheck(Version(13, 0, 0))` for Xcode 13.0.0.
         public let lastXcodeUpgradeCheck: Version?
 
+        /// Allows to render markdown files inside the workspace including an .xcodesamples.plist inside it.
+        public let renderMarkdownReadme: Bool
+
         public init(
             enableAutomaticXcodeSchemes: Bool?,
             autogeneratedWorkspaceSchemes: AutogeneratedWorkspaceSchemes,
-            lastXcodeUpgradeCheck: Version?
+            lastXcodeUpgradeCheck: Version?,
+            renderMarkdownReadme: Bool = false
```

Comment:
This is an internal type, so better to not add defaults, so we are sure we don't forgot to pass the value around